project-template/{{cookiecutter.repo_name}}/noxconfig.py

```diff
@@ -17,8 +17,8 @@ class Config:
         / "version.py"
     )
     path_filters: Iterable[str] = ()
-    pyupgrade_args = ("--py{{cookiecutter.python_version_min | replace('.', '')}}-plus",)
-    plugins = []
+    pyupgrade_args: Iterable[str] = ("--py{{cookiecutter.python_version_min | replace('.', '')}}-plus",)
+    plugins: Iterable[object] = ()
```

Comment:
yeah, that I encountered also during an dependency update in the kehrwoche, good that it is in the template